httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/GGSSchemeBase.java

```diff
@@ -138,7 +236,9 @@ protected byte[] generateGSSToken(
         final GSSManager manager = getManager();
         final GSSName serverName = manager.createName(serviceName + "@" + authServer, GSSName.NT_HOSTBASED_SERVICE);
```

Comment:
Well, that is not a server name, but a SPN...

Reply:
This is unchanged, but I will rename it.

Reply:
What would you prefer ? 'peer' ? 'principal' ? 'spn' ?

Reply:
peerName sounds reasonable and it also reflects the same term in the TLS code, i.e., peer certificate.

Reply:
I went with spn for now.

Reply:
renamed to peerName

Reply:
> I went with spn for now.

SPN is a Kerberos term, not GSS-API. `peerName` is just acceptable.